libs/libvtrutil/src/vtr_geometry.h

```diff
@@ -67,8 +67,10 @@ class Point {
 template<class T>
 class Rect {
   public: //Constructors
+    Rect();
```

Comment:
The `Rect` changes should be separated into a PR for upstream VTR.

Reply:
I've submitted a PR upstream: https://github.com/verilog-to-routing/vtr-verilog-to-routing/pull/1025/